testproject/Assets/Tests/Manual/Scripts/NetworkPrefabPool.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections;
```

Comment:
I think this is a bit over complicated because we expect this pool to be part of a scene and loaded/ unloaded with the scene. It's probably fine here in the test project.